ypkg2/ypkgcontext.py

```diff
@@ -242,6 +242,12 @@ def get_path(self):
             if os.path.exists(i):
                 console_ui.emit_info("Build", "Enabling ccache")
                 return "{}:{}".format(i, default_path)
+
+        # If ccache is enabled, sccache is also enabled. However, sccache
+        # doesn't need to manipulate PATH, so only a log is emitted. No further
+        # action is done
+        console_ui.emit_info("Build", "Enabling sccache")
```

Comment:
Can you check if sscache exists in path first before emitting this message. Otherwise sscache can be "enabled" when sscache isn't installed.

Note to itself: ypkg should probably emit a warning if sccache/ccache is enabled but sscache/ccache is not found in path.

Reply:
Done. I did checking in both places. The logic might feel a bit redundant, but for consistency I decided to emit the warning message at the same place where `ccache` is enabled/disabled. The actual disabling of `sccache` is at `ypkg2/scripts.py` L180. Since they use the same checking logic, they should have the same outcome.